lib/storage/file.js

```diff
@@ -1019,12 +1019,22 @@ File.prototype.getSignedPolicy = function(options, callback) {
  *     link will expire.
  * @param {string=} options.extensionHeaders - If these headers are used, the
  *     server will check to make sure that the client provides matching values.
+ * @param {string=} options.promptSaveAs - If you provide this value the SaveAs 
+ *     prompt will be displayed when accessing the signed url and the file name 
+ *     will be set to the provided value. If options.responseDisposition is set 
+ *     this option is ignored.
+ * @param {string=} options.responseDisposition - If you provide this value, 
```

Comment:
Seems wordy. Might be your style of param descriptions but I'd just say what the parameter represents, not really the consequences of setting it... that is implied through its usage. Something like: "The filename to prompt the user to save the file as when the signed url is accessed. This is ignored if options.responseDisposition is set."